engines/griffon/cutscenes.cpp

```diff
@@ -234,15 +259,23 @@ void GriffonEngine::intro() {
 			y--;
 		}
 
-		for (int i = 0; i <= 37; i++) {
+		for (int i = 0; i < ARRAYSIZE(story); i++) {
 			int yy = y + i * 10;
+
+#ifdef USE_TTS
+			if (i == nextparagraph)
+				nextparagraph = textToSpeech(nextparagraph, story, ARRAYSIZE(story));
+#endif
+
 			if (yy > -8 && yy < 240) {
 				int x = 160 - strlen(story[i]) * 4;
 				drawString(_videoBuffer, story[i], x, yy, 4);
 			}
-
-			if (yy < 10 && i == 37)
-				return;
+			if (yy < 10 && i == ARRAYSIZE(story) - 1) {
+				if (ttsMan != nullptr)
+					if (ttsMan->isSpeaking() == false)
+						return;
+			}
```

Comment:
That looks strange. Now we return only if `ttsMan` is not nullptr, and `ttsMan->isSpeaking()` is false. It seems to me that it should return in the case where `ttsMan` is nullptr.
So I would expect a test like:
```c++
if (yy < 10 && i == ARRAYSIZE(story) - 1) {	
	if (ttsMan == nullptr || ttsMan->isSpeaking() == false)
		return;
}
```